src/data/tag.ts

```diff
@@ -50,26 +51,16 @@ export function updateTag(tag: Tag, store = tagsStore) {
   }
   tags[iToUpdate] = tag;
   store.set(tags);
-  saveTags(store);
+  saveTags();
 }
 
 export function updateAllTags(visible: boolean, store = tagsStore) {
   const tags = get(store);
   tags.forEach((t) => (t.visible = visible));
   store.set(tags);
-  saveTags(store);
+  saveTags();
 }
 
-export function saveTags(store = tagsStore) {
-  const tags = get(store);
-  localStorage.setItem('tags', JSON.stringify(tags));
-}
-
-export function loadTags(store = tagsStore) {
-  const jsonString = localStorage.getItem('tags');
-  if (jsonString) {
-    store.set(JSON.parse(jsonString));
-  } else {
-    console.error('Failed to load tags from localStorage');
-  }
+export function saveTags(tags = tagsStore) {
```

Comment:
I could remove this function to just add `saveResumeDataToLocalStorage`, but that would mean I need to change all of the tests. I can save that for a later time.